src/main.rs

```diff
@@ -1,7 +1,12 @@
 use clap::Parser;
-use project_cleaner::{args::Args, ui::run};
+use project_cleaner::{args::*, ui, simple};
```

Comment:
why use `*` when only `Args` is needed?

Reply:
Oh, that's a minor oversight from when I was planning to move `AppResult` typedef into `args` module to use it in my code as well, but as it turned out, it wasn't necessary and the import stayed as I originally wrote it...